astronomer/providers/snowflake/extractors/snowflake_async_extractor.py

```diff
@@ -0,0 +1,162 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Do we need this license?

Reply:
Thought of removing but it was taken from openlineage and we changed few things as per requirements